src/utils/bundleReport.js

```diff
@@ -0,0 +1,106 @@
+const path = require('path');
+const sizeTypes = ['B', 'KB', 'MB', 'GB'];
```

Comment:
Should `KB` be `kB`?

Reply:
U are correct it should be kibiByte (kB)